composer.json

```diff
@@ -14,7 +14,7 @@
         }
     },
     "require": {
-        "php": ">=7",
+        "php": ">=7.2",
```

Comment:
already PHP8 compatible? maybe use ^ instead?

Reply:
no release, no compatibility. There are still undocumented changes, so I would like to wait. The phar is, btw not limited here